charter.html

```diff
@@ -0,0 +1,285 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>
+      Color-on-the-web Community Group Charter
+    </title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="//www.w3.org/StyleSheets/TR/base">
+    <style>
+      body {
+        max-width: 60em;
+        margin: auto;;
+      }
+      *:target {
+        background-color: yellow;
+      }
+      li {
+        margin-bottom: 9pt;
+      }
+      .note {
+        background-color: yellow;
+        padding: 10px;
+      }
+      .remove {
+        background-color: yellow;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>
+      [DRAFT] Color-on-the-web Community Group Charter
+    </h1>
+    <p>
+      This Charter is work in progress. To submit feedback, please use <a
+      href="https://github.com/w3c/ColorWeb-CG/issues">https://github.com/w3c/ColorWeb-CG/issues</a>
+      where the Charter is being developed.
+    </p>
+    <ul>
+      <li>This Charter: <i>n/a</i>
+      </li>
+      <li>Previous Charter: <i>n/a</i>
+      </li>
+      <li>Start Date:  <time datetime="2017-01-27">Fri, 27 Jan 2017</time></span>
+      </li>
+      <li>Last Modified: <i>n/a</i></span>
+      </li>
+    </ul>
+    <h2 id="goals">
+      Goals
+    </h2>
+    <p>
+      The goal is to allow color experts from various fields - including
+      print-oriented color management, display, media and entertainment
+      community, CSS , image and video coding and graphics experts - to come
+      together, share ideas, use cases, experiences and discuss technical
+      solutions to improve the state of color on the Web.
+    </p>
+    <h2 id="scope-of-work">
+      Scope of Work
+    </h2>
+    <p>The works addresses the representation and processing of color
+    information as applied to the Web.</p>
+    <p>
+      Particular areas of interest include, but are not limited to:
+      <ul>
+        <li>Unambiguous, colorimetric color specification of desired presentation</li>
+        <li>Wide gamut colorspaces (wider than sRGB)</li>
+        <li>High dynamic range (greater range of luminance, darker blacks and
+        peak whites greater than paper white)</li>
+        <li>Color matching between media (raster images, video, vector graphics)
+        and text/styling content (styled by CSS)</li>
+        <li>Consistency and predictability of display</li>
+      </ul>
+    </p>
+    <h3 id="out-of-scope">
+      Out of Scope
+    </h3>
+    <p><i>n/a</i></p>
```

Comment:
Can we drop this section? This makes it seem like everything could potentially be in scope.

Reply:
Instead of dropping it, I'd rather it contain a concise list of things that are definitely out of scope, e.g.

* anything unrelated to color
* native platform color things unrelated to the web

or some such.

Reply:
The scope is _The works addresses the representation and processing of color information as applied to the Web_, which excludes topics unrelated to the intersection of web and color.

Do you have very specific examples of topics that would be out-of-scope.

Reply:
The "Out of Scope" section in charters is for things that look like they might be in scope, but are not. For example because another group or org is already doing that. As an example the Web Fonts WG specifically excludes changes to OFF/OpenType, which are instead done by liaison to SC29.

If there is nothing in that category (as here) then the section is simply deleted.